script.js

```diff
@@ -1,4 +1,4 @@
-const DEFAULT_PORT = 22023;
+const DEFAULT_PORT = 22000;
```

Comment:
Why `22000` and not `80`?

Reply:
22000 is the default port that Impostor.Http uses and therefore the port that people who don't use a reverse proxy will use.